Week2task2/src/Main.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+public class Main {
+
+	public Main() {
+		// TODO Auto-generated constructor stub
+	}
+
+	public static void main(String[] args) {
+		Name person = new Name();
+		person.name = "Anna";
+		person.surname = "Wick";
```

Comment:
Use the constructor that has as params name and surname. person = new Name("Wick", "Anna").